DM-31193: Add note about detectors 2,61 to readme (#29)
lsst/ap_verify_hits2015: README.md

This repository contains over 80 DECam images with 60+ CCDs each! If you want a smaller dataset to practice running `ap_verify`, consider `ap_verify_ci_hits2015` or `ap_verify_ci_cosmos_pdr2`.

> Due to problems with detectors 2 and 61, this repository does not contain any data or calibrations for those detectors. Because of this, you must specify this data-query argument when running ap_verify on this data:

Reviewer: Can you double-check this? `butler query-datasets`, at least, thinks there are both raws and calibs with these detectors.

Author: That's what this thread was about:

https://lsstc.slack.com/archives/C2B6X08LS/p1633655323064500

If I don't skip them, the task won't even start.

This repository contains over 80 DECam images with 60+ CCDs each! If you want a smaller dataset to practice running `ap_verify`, consider `ap_verify_ci_hits2015` or `ap_verify_ci_cosmos_pdr2`.

> Due to problems with detectors 2 and 61, this repository does not contain all the necessary data to process these detectors.

Reviewer: I'm not convinced the data are really missing, rather than e.g. corrupted. Maybe:
```suggestion
Due to problems with detectors 2 and 61, this repository cannot process these detectors.
```
